### PR TITLE
Add option to disable font override (#556)

### PR DESCRIPTION
**Pull request by pablopunk**
I agree with https://github.com/sindresorhus/refined-github/issues/556 cause I prefer the new default SF Mono than Fira Code (which was the one I had with the extension)

### CODE REVIEW

**Comment by pablopunk** [comment]
@busches the current css rule would still be there as the default. If the user unchecks that option, it would just override them with the class `default-fonts`.

I'm not sure what you mean by the font order from the current rule, I didn't change it.

**Comment by busches** [comment]
Ok, I think I misunderstood the implemention. You're allowing the override
and then essentially undoing it? Wouldn't it be easier to only apply the
font override when the option is checked? Right now if GitHub changes the
default font options we'd have to make code changes too.

**Comment by pablopunk** [comment]
Yeah, I guess that makes more sense. I'll just add a `.override-fonts` class that would only be added when the option is selected.

File: src/content.js [hunk 1]
```diff
@@ -452,6 +452,24 @@ function addTitleToEmojis() {
...
+	.blob-code-inner,
+	.export-phrase pre,
+	.file-editor-textarea,
+	.gollum-editor .expanded textarea,
+	.gollum-editor .gollum-editor-body,
+	.input-monospace,
+	.wiki-wrapper .wiki-history .commit-meta code`)
+	.addClass('override-fonts');
+}
+
```
Comment (bfred-it) [inline src/content.js]: That's a lot of operations. I'd
1. leave this in the CSS
2. prefix each selector with `.override-fonts`
3. Only add the `override-fonts` class to `<html>` via JS

Comment (pablopunk) [inline src/content.js]: hmm, that seems better for performance, done!

**Comment by sindresorhus** [comment]
Note my comment in #556 

> Either we drop it altogether or we disable it by default and add a font and font size option.

**Comment by jdreesen** [comment]
I'd like to have the option to use Fira Code as font.

**Comment by bfred-it** [comment]
@sindresorhus if it's disabled by default and we leave the choice to the user, should it be part of Refined GitHub at all? How would that be different from just leaving it to a stylesheet?

**Comment by sindresorhus** [comment]
@bfred-it I think it would make more sense as a separate extension that works on code blocks on any website, not just GitHub.

**Comment by bfred-it** [comment]
Perfect, then:

**Comment by pablopunk** [comment]
I guess we should close this then?

**Pull request closed without merging**